Tools/SystemDebug/docs/EFIApplication.rst

```diff
@@ -89,7 +89,9 @@ build Process
 
 To build the efi applcation, please clone the repository.
 
-``git clone https://github.com/intel/IoTKit-code-samples``
+``git clone https://github.com/oneapi-src/oneAPI-samples.gits``
```

Comment:
I think there is a stray 's' at the end here?

Reply:
it looks fine